src/node_api.cc

```diff
@@ -492,15 +477,41 @@ class TryCatch : public v8::TryCatch {
 
 //=== Function napi_callback wrapper =================================
 
-static const int kDataIndex = 0;
-static const int kEnvIndex = 1;
+static const int kFunctionIndex = 0;  // Used in CallbackBundle::cb[]
+static const int kGetterIndex = 0;    // Used in CallbackBundle::cb[]
+static const int kSetterIndex = 1;    // Used in CallbackBundle::cb[]
+static const int kCallbackCount = 2;  // Used in CallbackBundle::cb[]
+                                      // Max is "getter + setter" case
+
+static const int kCallbackBundleIndex = 0;  // The first and the only one
+static const int kInternalFieldCount = 1;
 
-static const int kFunctionIndex = 2;
-static const int kFunctionFieldCount = 3;
+// Use this data structure to reduce the number
+// of GetInternalField() calls to only 1 (was: 3).
```

Comment:
nit: We no longer access this data via `GetInternalField()`. How about we change this comment to
```C++
// Use this data structure to associate callback data with each N-API function
// exposed to JavaScript. The structure is stored in a v8::External which gets
// passed into our callback wrapper. This reduces the performance impact of
// calling through N-API.
// Ref: benchmark/misc/function_call
```

Reply:
Yep, I did forget to update the comments, my bad.